### ISSUE TITLE
Missing typespec for Lens.into

### ISSUE THREAD

**Issue by lkuty** [obrok/lens#14]
In the source code of `lens.ex`, the definition of `into` is missing a typespec. That is the cause of a dialyxir error.

```elixir
  deflens_raw into(lens, collectable) do
    fn data, fun ->
      {res, updated} = get_and_map(lens, data, fun)
      {res, Enum.into(updated, collectable)}
    end
  end
```

```
Function call without opaqueness type mismatch.

Call does not have expected opaque term of type Lens.t() in the 2nd position.

Common.Utilities.lens_map(
  any(),
  _lens4 :: (:get | :get_and_update, _, (_ -> any()) -> any()),
  ...
```

**Comment by obrok** [comment]
Should be fixed by fde56e4f647d4de220e86597cf15be9c280fe133, could you check?

**Comment by lkuty** [comment]
That's ok now. Thanks
